Projects (#227)
OCR-D/ocrd-website: site/en/_posts/2021-06-10-projects.md

Of the eleven proposals submitted, which were developed in the course of extensive piloting 
of the OCR-D software in the summer of 2020, four implementation and three module projects 
will be funded by the DFG. 

The module projects will further develop selected OCR-D tools.
* Workflow for work-specific training based on generic models with OCR-D as well as ground truth enhancement (UB Mannheim).
* Font Group Recognition for Improved OCR (JGU Mainz, FAU Erlangen)
* OLA-HD Service - A Generic Service for Long-Term Archiving of Historical Prints (SUB Göttingen, GWDG)

Together with the ca. 65 other OCR-D tools,  these form the basis for the work of the four implementation projects, which will prepare the existing OCR-D software for its productive use in mass digitisation. 
* Integration of Kitodo and OCR-D for productive mass digitisation (UB Braunschweig, SLUB Dresden, UB Mannheim).
* OPERANDI: OCR-D Performance Optimisation and Integration (SUB Göttingen, GWDG)
* OCR-D Software in Modular Mass Digitisation Workflows (ULB Halle)
* OCR4all libraries full text recognition of historical collections (GEI Braunschweig, HCI and ZPD of the University of Würzburg)

> We are very much looking forward to the ( further ) cooperation with the implementation and module projects, which will prepare the use of the OCR-D software in different usage scenarios, and are excited about this next project phase.

Reviewer: Leerzeichen nach und vor "further" weg.

---
layout: post
lang: en
lang-ref: Bewilligung OCR-D-Projekte
title: "Implementation and module projects granted"
tags: [DFG, grant, phase 3]
author: Elisabeth Engl
---

> In addition to the coordination project proposal, the DFG also approved seven implementation 

Reviewer 2: ```suggestion
In addition to the coordination project, the DFG also approved seven implementation 
```

and module projects that will begin their work in the coming months.

Of the eleven proposals submitted, which were developed in the course of extensive piloting 
of the OCR-D software in the summer of 2020, four implementation and three module projects 
will be funded by the DFG. 

The module projects will further develop selected OCR-D tools.
* Workflow for work-specific training based on generic models with OCR-D as well as ground truth enhancement (UB Mannheim).
* Font Group Recognition for Improved OCR (JGU Mainz, FAU Erlangen)

> * OLA HD Service - A Generic Service for Long-Term Archiving of Historical Prints (SUB Göttingen, GWDG)

Reviewer 2: ```suggestion
* OLA-HD Service - A Generic Service for Long-Term Archiving of Historical Prints (SUB Göttingen, GWDG)
```

> Together with the other OCR-D tools, which currently number around 65, these form the basis for the work of the four implementation projects, which will prepare the existing running OCR-D prototype for its productive use in mass digitisation. 

Reviewer 2: ```suggestion
Together with the ca. 65 other OCR-D tools,  these form the basis for the work of the four implementation projects, which will prepare the existing OCR-D software for its productive use in mass digitisation. 
```
